Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionSetsWindow/MotionSetsWindowPlugin.cpp

```diff
@@ -139,20 +135,20 @@ namespace EMStudio
         m_dirtyFilesCallback             = nullptr;
     }
 
-
-    // destructor
     MotionSetsWindowPlugin::~MotionSetsWindowPlugin()
     {
-        GetCommandManager()->RemoveCommandCallback(m_createMotionSetCallback, false);
-        GetCommandManager()->RemoveCommandCallback(m_reinitCallback, false);
+        for (MCore::Command::Callback* callback : m_callbacks)
+        {
+            GetCommandManager()->RemoveCommandCallback(callback, false);
+            delete callback;
+        }
+
         GetCommandManager()->RemoveCommandCallback(m_adjustMotionSetCallback, false);
```

Comment:
Is it too much to ask to put other callback also in the m_callbacks array for consistency?

Reply:
Will change, good call.